manual/src/cmds/ocamldep.etex

```diff
@@ -149,55 +151,63 @@ Display a short usage summary and exit.
 %
 \end{options}
 
-\section{s:ocamldep-makefile}{A typical Makefile}
+\section{s:ocamldep-makefile}{A Makefile}
 
-Here is a template "Makefile" for a OCaml program.
+Here is an example "Makefile" for a OCaml program in order to
+demonstrate the contribution of ocamldep to the compilation model.
```

Comment:
I like the change from `template` to `example`.

I am not sure what `contribution of ocamldep to the compilation model` means. Did you mean something like:

> in order to demonstrate how to use ocamldep to compute dependencies between modules.

That still seem pretty redundant to the rest of the text.

Reply:
Agreed it's redundant, updated to just change "template" to "example"